libtelemetry/src/main/java/com/mapbox/services/android/telemetry/MapboxTelemetry.java

```diff
@@ -96,6 +97,18 @@ public void updateSessionIdRotationInterval(int hour) {
     }
   }
 
+  public boolean isDebugLoggingEnabled() {
```

Comment:
Why are we exposing `isDebugLoggingEnabled()`? If not needed, let's remove it.

Reply:
Per above description and associated tickets, this is a public api so other sdks and apps can utilize the additional logging to help with testing.